### ISSUE TITLE
Save corruption in Dragonball Fusions/Triforce Heroes after connecting debugger

### ISSUE THREAD

**Issue by iSharingan** [imthe666st/NTRClient#30]
So, when using both cartridge and SD/Cia installed versions connecting the debugger will make any saves made/loaded register as corrupt both during and after debugger use (even if you simply connect then immediately disconnect the debugger)

JKSM returns an error when trying to read the save archives after this happens (can be remedied by letting the game format the save archive then restoring a save backup, but all progress since that save is, ofc, lost) so it's not just randomizing a couple bytes in the save.

This issue is the same as what I'm posting to 44670's master in case this version is no longer maintained

**Comment by imthe666st** [comment]
I'm no longer working on this fork of the NTR debugger. 44670 isn't working on the original anymore either. 
I'm not quite sure why it happens though. Does this also apply to other games?

**Comment by iSharingan** [comment]
I have noticed games freezing/crashing when the debugger is connected too close to a save/load operation, but those are the only 2 thus far to outright damage save files/actions.

Kingdom Hearts DDD would say "unable to save" while the debugger was connected, but it later works fine after it disconnects (even if still streaming to NTR viewer, though the issue also persists on NTR 3.2/3.2. Also, skipping the debugger and just using kitkat to stream doesnt cause the issue - though finding offsets for use in codes isnt an option there).

**Comment by iSharingan** [comment]
Just to update, I found another game: LBX Little Battlers eXperience. Fourtunately KitKat allows me to stream without issue. After testing codes via the this tool then converting them into NTR plugins, The memory locations are not changed at all. I suspect the issue might actually be related to redirecting RAM over a wifi connection, as similar issues have been mentioned in a TWL-mode tool to access old DS games from a SD card (redirecting the save location to the SD while an actual cartridge is inserted to get into TWL mode, as I understand it.)

Long story short, this only seems to affect games whose physical-cartridge versions would use a particular type of memory for save data - hence why only some 3ds games are affected (these games were released around the same time, which suggests the same hardware in the cartridges)

**Comment by iSharingan** [comment]
further update: Using the 1.6 release from May 10th (after the nightly releases) doesn't cause the error with the saves for Triforce heroes or LBX (the issue remains for Dragon Ball Fusions). Still not quite sure why, but I'll see if I can find any possible causes in the commits since that release if/when I have time.
